packages/client-sdk-nodejs/src/internal/grpc/middlewares-interceptor.ts

```diff
@@ -75,6 +77,17 @@ export function middlewaresInterceptor(
             status: StatusObject,
             next: (status: StatusObject) => void
           ): void {
+            if (status.code === 4) {
```

Comment:
there is a constant we can use that is like DEADLINE_EXCEEDED or something, I believe, rather than hard-coding the 4 here.